app/javascript/src/components/Dashboard/Notes/ListNoteCards.jsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+
+import Card from "components/Dashboard/Notes/Card";
+
+import { calculateAge, formatTime } from "./utils/prettyPrintTime";
```

Comment:
No need to add another file called `prettyPrintTime` yet. Currently we only have a few utils in hand. It's fine to keep them all in a single `utils.js` rather than prematurely splitting them into multiple files. We don't know whether in future this `utils.js` file become bloated or not. When it becomes bloated, let's work towards splitting it up.